thali/thalireplicationmanager.js

```diff
@@ -6,6 +6,7 @@ var inherits = require('util').inherits;
 var net = require('net');
 var multiplex = require('multiplex');
 var validations = require('./validations');
+var addressPrefix = 'addressbook-'
```

Comment:
Why is addressPrefix in the thalireplicationmanager? That is something that is specific to the postcardapp. It is not generic to Thali at all. It shouldn't be Thali_CordovaPlugin, it should be in postcardapp.